adminutils/adminutils.py

```diff
@@ -253,7 +292,7 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
             await ctx.send_help()
             return
         try:
-            await emoji.edit(
+            em = await emoji.edit(
```

Comment:
Seems like `emoji.edit` doesnt return anything, you need to re-get emoji, e.g. via `em = bot.get_emoji(emoji.id)`

https://discordpy.readthedocs.io/en/stable/api.html#discord.Emoji.edit